CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Added
 - Testing for Ruby 2.4.1
 
+# [1.2.1] - 2017-09-24
```

Comment:
We do versioning on deploy, you can read about our changelog conventions [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

Reply:
Thanks for getting back to me on that @majormoses and sorry for the long delay. Since I am still new to contributing to upstream, I just want to make clear what I need to do :).

- I need to add examples of it in action (testing artifact)
- I don't need to do anything with the version as that is handled on deploy. So my addition to the change log should be more like:

```
# [Unreleased]
### Added
- Added options to allow for different age limits when the agent is disabled
- Added a switch to use the new limits so as not to impact current functionality
```

Thanks again for getting back to me on that and hope I am on the right track but let me know if I missed anything

Reply:
Oh man sorry I missed this and it fell off my radar, yes you are correct. I would love to see this fixed up and over the finish line.